### DISCUSSION TITLE
How to change defaultTagBackgrounds to my custom colour?

### DISCUSSION THREAD

**Discussion by wojciehm** [CaiJimmy/hugo-theme-stack#1080] [Q&A] [answered]
Hey! First of all thanks for awesome theme @CaiJimmy 

I wanted to do a simple change to the colours of the categories. I quickly found that they are defined in variables.scss.

I changed the default ones from

```
$defaultTagBackgrounds: #8ea885, #df7988, #0177b8, #ffb900, #6b69d6;
```
to my custom one which is 
```
$defaultTagBackgrounds: #33a5dd;
```
I placed the file in ```layouts\assets\scss``` folder

<img width="436" alt="image" src="https://github.com/user-attachments/assets/0a984267-0c6b-45a1-b07d-c111e94c9e0e">

Change isn't reflected on my blog. 

<img width="1083" alt="image" src="https://github.com/user-attachments/assets/b9cd9e61-820f-4fd2-9b8f-fbcd7695d7cd">

What I am doing wrong here?

TiA

BR,
Wojciech

**Comment by CaiJimmy** [comment, marked as answer]
Should be `assets\scss` folder instead

**Comment by wojciehm** [comment]
Thanks @CaiJimmy . Worked like a charm.

<img width="564" alt="image" src="https://github.com/user-attachments/assets/2d2fc863-7351-4114-a637-9c685cfd9553">
